# **Assessment for Applied Statistics module**

**by Matthias Wiedemann**

Student at [ATU](https://www.atu.ie/).

<img src="https://studenthub.atu.ie/assets/ATU_Logo.fa93bf0a.svg" alt="ATU Logo" width="300" height="100">

The assessment instructions can be found [here]( https://github.com/ianmcloughlin/applied-statistics/blob/main/assessment/assessment.md).

## Problem 1: Extending the Lady Tasting Tea

In the original Fisher’s Lady Tasting Tea experiment there are 8 cups total: 4 tea-first and 4 milk-first. 

The lady claims she can distinguish them.

What we have learned in the [lecture](https://github.com/ianmcloughlin/applied-statistics/blob/main/materials/lady-tasting-tea.ipynb) is that if she guesses all 8 correct, the probability (the p-value if she gets all 8 correct under the null hypothes is) of doing that by chance is:

$$
P = \frac{1}{\binom{8}{4}} = \frac{1}{70} \approx 0.0143
$$

For the extended experiment, 12 cups total, 8 tea-first and 4 milk-first we need to recalculate the Binomial Coefficient, ("N choose K)," and will use the same code as shown in the lecture:

1. First of all we import the relevant libraries:

In [4]:
# Mathematical functions from the standard library.
# https://docs.python.org/3/library/math.html
import math

# Permutations and combinations.
# https://docs.python.org/3/library/itertools.html
import itertools

# Random selections.
# https://docs.python.org/3/library/random.html
import random

# Numerical structures and operations.
# https://numpy.org/doc/stable/reference/index.html#reference
import numpy as np

# Plotting.
# https://matplotlib.org/stable/contents.html
import matplotlib.pyplot as plt

2. Cups of tea

In [5]:
# Number of cups of tea in total.
no_cups = 12

# Number of cups of tea with milk in first.
no_cups_milk_first = 4

# Number of cups of tea with tea in first.
no_cups_tea_first = 8



3. We use math.comb(n, k) to calculate the number of ways to choose k elements from a set of n elements where the order of the chosen elements does not matter and no element is selected more than once:

In [6]:
# Number of ways of selecting four cups from eight.
ways = math.comb(no_cups, no_cups_milk_first)

# Show.
ways

495

The total number of unique ways to choose 4 items from 12 is 495.

Let's apply this value to the extended experiment, 12 cups total: 8 tea-first and 4 milk-first:

$$
P = \frac{1}{\binom{12}{4}} = \frac{1}{495} \approx 0.00202
$$

https://www.statology.org/p-values-explained-in-plain-english-with-visuals/
https://www.geeksforgeeks.org/machine-learning/p-value/

## Problem 2: Normal Distribution

## Problem 3: t-Tests

## Problem 4: ANOVA

### End.